### PR TITLE
br: add a document for backup restore systables

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add a document for backup restore system tables.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -89,6 +89,16 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份恢复
+
+BR (v5.1.0 之前) 备份时会过滤掉系统库表的数据。
+BR (v5.1.0 之后) 默认会备份全部数据，包括系统库(mysql)，但是恢复的时候默认*不*恢复系统表，只有设置了 filter 参数才会恢复到临时库中，然后通过 rename 的方式恢复到系统库。这样是为了兼容之前 BR 的版本。
```
Comment (kennytm) [inline br/backup-and-restore-tool.md:95]:
```suggestion
BR (v5.1.0 之后) 默认会备份全部数据，包括系统库 (`mysql.*`)，但是恢复的时候默认*不*恢复系统表，只有设置了 filter 参数才会恢复到临时库中，然后通过 rename 的方式恢复到系统库。这样是为了兼容之前 BR 的版本。
```

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -89,6 +89,16 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份恢复
+
+BR (v5.1.0 之前) 备份时会过滤掉系统库表的数据。
```
Comment (kennytm) [inline br/backup-and-restore-tool.md:94]:
```
br/backup-and-restore-tool.md: this file has manual line breaks in the following lines:

MANUAL LINE BREAKS: L95
MANUAL LINE BREAKS: L96
```

File: br/backup-and-restore-tool.md [hunk 3]
```diff
@@ -89,6 +89,16 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
+BR (v5.1.0 之后) 默认会备份全部数据，包括系统库(mysql)，但是恢复的时候默认*不*恢复系统表，只有设置了 filter 参数才会恢复到临时库中，然后通过 rename 的方式恢复到系统库。这样是为了兼容之前 BR 的版本。
+同时对于以下系统库表进行特殊处理:
+
+- 统计信息相关的表: 不进行恢复，因为统计信息的 table id 发生了变化。
+- mysql 下的 tidb/global_variables 表: 不进行恢复，因为该表不能覆盖，如 gc safe point 覆盖后会对集群产生影响。
```
Comment (kennytm) [inline br/backup-and-restore-tool.md:99]:
```suggestion
- mysql 库下的 tidb 和 global_variables 表: 不进行恢复，因为该表不能覆盖，如 gc safe point 覆盖后会对集群产生影响。
```

File: br/backup-and-restore-tool.md [hunk 4]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
+
+BR (v5.1.0 之前) 备份时会过滤掉系统库表的数据。
+
+BR (v5.1.0 之后) 默认会备份全部数据，包括系统库 (`mysql.*`)，但是恢复的时候默认*不*恢复系统表，只有设置了 filter 参数才会恢复到临时库中，然后通过 rename 的方式恢复到系统库。这样是为了兼容之前 BR 的版本。
+
+同时对于以下系统库表进行特殊处理:
+
+- 统计信息相关的表: 不进行恢复，因为统计信息的 table id 发生了变化。
+- mysql 库下的 tidb 和 global_variables 表: 不进行恢复，因为该表不能覆盖，如 gc safe point 覆盖后会对集群产生影响。
+- mysql 下的 user 表: 恢复后需要手动执行 FLUSH PRIVILEGE 才能生效。
```
Comment (kennytm) [inline br/backup-and-restore-tool.md:102]:
```suggestion
- mysql 库下的 user 表: 恢复后需要手动执行 FLUSH PRIVILEGE 才能生效。
```

**Review by kennytm — COMMENTED** [review]
rest LGTM

File: br/backup-and-restore-tool.md [hunk 5]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份恢复
+
+BR (v5.1.0 之前) 备份时会过滤掉系统库表的数据。
+
+BR (v5.1.0 之后) 默认会备份全部数据，包括系统库 (`mysql.*`)，但是恢复的时候默认*不*恢复系统表，只有设置了 filter 参数才会恢复到临时库中，然后通过 rename 的方式恢复到系统库。这样是为了兼容之前 BR 的版本。
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:96]:
```suggestion
自 v5.1.0 起，BR 默认会**备份**全部数据，包括系统库 (`mysql.*`)，但为了兼容之前 BR 的版本，**恢复**的时候默认**不**恢复系统表，只有设置了 [`filter` 参数](https://docs.pingcap.com/zh/tidb/stable/use-br-command-line-tool#使用表库过滤功能备份多张表的数据)才会把系统表恢复到临时库中，然后通过 rename 的方式恢复到系统库。
```
斜体预览看起来好像不是特别明显，所以改成了加粗。

这个文件是 BR 的概览文件，可能是用户想了解 BR 时会首先接触的文档，这里里面提到的 `filter` 参数和 rename 方式似乎在上文都没有提及，是不是属于更高级的内容？如果是的话对新手用户感觉不太友好。

有没有相应的网页链接可以加一下？方便用户进一步了解

File: br/backup-and-restore-tool.md [hunk 6]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
+
+同时对于以下系统库表进行特殊处理:
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:98]:
```suggestion
同时，对于下列系统库表会进行特殊处理:
```

File: br/backup-and-restore-tool.md (hunk 4 above)
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:102]:
```suggestion
- 统计信息相关的表：不进行恢复，因为统计信息的 table id 发生了变化。
- mysql 库下的 `tidb` 和 `global_variables` 表：不进行恢复，因为该表不能覆盖，例如 GC safepoint 覆盖后会对集群产生影响。
- mysql 下的 `user` 表：恢复后需要手动执行 `FLUSH PRIVILEGE` 才能生效。
```

File: br/backup-and-restore-tool.md [hunk 7]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份恢复
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:92]:
```suggestion
### 系统库表的备份与恢复
```

File: br/backup-and-restore-tool.md [hunk 8]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
+
+BR (v5.1.0 之前) 备份时会过滤掉系统库表的数据。
+
+BR (v5.1.0 之后) 默认会备份全部数据，包括系统库 (`mysql.*`)，但是恢复的时候默认*不*恢复系统表，只有设置了 filter 参数才会恢复到临时库中，然后通过 rename 的方式恢复到系统库。这样是为了兼容之前 BR 的版本。
+
+同时，对于下列系统库表会进行特殊处理:
+
+- 统计信息相关的表: 不进行恢复，因为统计信息的 table id 发生了变化。
+- mysql 库下的 tidb 和 global_variables 表: 不进行恢复，因为该表不能覆盖，如 gc safe point 覆盖后会对集群产生影响。
+- mysql 库下的 user 表: 恢复后需要手动执行 FLUSH PRIVILEGE 才能生效。
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:102]:
```suggestion
同时，对于下列系统库表会进行特殊处理：

- 统计信息相关的表：不进行恢复，因为统计信息的 table id 发生了变化。
- mysql 库下的 `tidb` 和 `global_variables` 表：不进行恢复，因为该表不能覆盖，例如 GC safepoint 覆盖后会对集群产生影响。
- mysql 库下的 `user` 表：恢复后需要手动执行 `FLUSH PRIVILEGE` 才能生效。
```

File: br/backup-and-restore-tool.md [hunk 9]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份与恢复
+
+BR (v5.1.0 之前) 备份时会过滤掉系统库表的数据。
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:94]:
```suggestion
在 v5.1.0 之前，BR 备份时会过滤掉系统库表的数据。
```

File: br/backup-and-restore-tool.md (hunk 5 above)
Comment (3pointer) [reply br/backup-and-restore-tool.md]: 这里的 rename 指的是用 SQL rename table 的方式，🤔️，这里可以更明确下

**Review by CharLotteiu — APPROVED** [review]
LGTM

File: br/backup-and-restore-tool.md [hunk 10]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份与恢复
+
+在 v5.1.0 之前，BR 备份时会过滤掉系统库表的数据。
+
+自 v5.1.0 起，BR 默认会**备份**全部数据，包括系统库 (`mysql.*`)，但为了兼容之前 BR 的版本，**恢复**的时候默认**不**恢复系统表，只有设置了 [`filter` 参数](https://docs.pingcap.com/zh/tidb/stable/use-br-command-line-tool#使用表库过滤功能备份多张表的数据)才会把系统表恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:96]:
```suggestion
自 v5.1.0 起，BR 默认会**备份**全部数据，包括系统库 (`mysql.*`)，但为了兼容之前 BR 的版本，**恢复**的时候默认**不**恢复系统表，只有设置了 [`filter` 参数](/use-br-command-line-tool.md#使用表库过滤功能备份多张表的数据)才会把系统表恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
```

File: br/backup-and-restore-tool.md [hunk 11]
```diff
@@ -89,6 +89,18 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
+### 系统库表的备份与恢复
+
+在 v5.1.0 之前，BR 备份时会过滤掉系统库表的数据。
+
+自 v5.1.0 起，BR 默认会**备份**全部数据，包括系统库 (`mysql.*`)，但为了兼容之前 BR 的版本，**恢复**的时候默认**不**恢复系统表，只有设置了 [`filter` 参数](/use-br-command-line-tool.md#使用表库过滤功能备份多张表的数据)才会把系统表恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
```
Comment (CharLotteiu) [inline br/backup-and-restore-tool.md:96]:
```suggestion
自 v5.1.0 起，BR 默认会**备份**全部数据，包括系统库 (`mysql.*`)，但为了兼容之前 BR 的版本，**恢复**的时候默认**不**恢复系统表，只有设置了 [`filter` 参数](/br/use-br-command-line-tool.md#使用表库过滤功能备份多张表的数据)才会把系统表恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
```

**Review by kennytm — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**